hack/build-venv.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Create and activate venv
+python -m venv .venv
+source .venv/bin/activate
```

Comment:
Might be easier to use [`tox`](https://tox.wiki/en/4.14.2/) for all of this?

For example - https://opendev.org/openstack/ansible-collections-openstack/src/branch/master/tox.ini

Reply:
Updated to use tox :+1: